sdk/identity/Azure.Identity/samples/DefiningCustomCredentialTypes.md

```diff
@@ -0,0 +1,128 @@
+# Defining Custom Credential types
+The Azure.Identity library covers a broad range of Azure Active Directory authentication scenarios. However, it's possible the credential implementations in Azure.Identity might not meet the specific needs your application, or an application might want to totally avoid a dependency on the Azure.Identity library.
```

Comment:
```suggestion
The Azure.Identity library covers a broad range of Azure Active Directory authentication scenarios. However, it's possible the credential implementations in Azure.Identity might not meet the specific needs your application, or an application might want to avoid taking a dependency on the Azure.Identity library.
```